lib/fizzy/execute.cpp

```diff
@@ -1553,8 +1557,10 @@ execution_result execute(
 
 end:
     assert(labels.empty() || trap);
-    // move allows to return derived Stack<uint64_t> instance into base vector<uint64_t> value
-    return {trap, std::move(stack)};
+    // move allows to return derived OperandStack instance into base vector<uint64_t> value
+    return {trap, stack.size() != 0 ?
+                      std::vector<uint64_t>{&stack[stack.size() - 1], &stack[0] + 1} :
```

Comment:
here as well would be useful to have
```
const uint64_t* OperandStack::cbegin() const;
const uint64_t* OperandStack::cend() const;
```

Reply:
I used `rbegin()`, `rend()` to be consistent with the indexing order, but I'm not confident that is the best. Please recheck.

Reply:
I think it's good.